### ISSUE TITLE
Digraph canonical labelling not respecting colours

### ISSUE THREAD

**Issue by james-d-mitchell** [gap-packages/Digraphs#19]
Originally reported by: **Finn Smith (Bitbucket: [flsmith](http://bitbucket.org/flsmith), GitHub: [flsmith](http://github.com/flsmith))**

---

I might have misunderstood the point behind this function, but I'm confused by the example that is given in the manual

```
#!GAP

gap> G := Digraph(10, [1, 1, 3, 4, 4, 5, 8, 8], [6, 3, 3, 9, 10, 9, 4, 10]);
<digraph with 10 vertices, 8 edges>

gap> DigraphCanonicalLabelling(G, [1, 1, 1, 1, 2, 3, 1, 3, 2, 1]);
(1,4,5,7)(3,6,9,8,10)
```

I can't see in what sense this result respects colouring.

In the different encoding of coloured digraphs I have, I get a different result... which also seems not to respect colouring.

---
- Bitbucket: https://bitbucket.org/james-d-mitchell/digraphs/issue/19

**Comment by wilfwilson** [comment]
I've convinced myself, and talked to @flsmith about it, that `DigraphCanonicalLabelling` is working correctly with respect to colours. However, I think that Finn is right to be confused by the documentation. I'll improve the documentation and then close this issue.

**Comment by wilfwilson** [comment]
The documentation essentially gave the impression that `DigraphCanonicalLabelling` for a coloured digraph returns a permutation that fixes the colouring.

**Comment by wilfwilson** [comment]
This was sorted out by #49, and released in `v0.7.0`.
